sentry-android-core/src/main/java/io/sentry/android/core/AndroidTransactionProfiler.java

```diff
@@ -307,6 +312,10 @@ public void close() {
           true,
           null,
           HubAdapter.getInstance().getOptions());
+    } else {
```

Comment:
```suggestion
    } else if (transactionsCounter != 0) {
```
should we check for it here?